nautobot/ipam/tests/test_migrations.py

```diff
@@ -421,7 +421,8 @@ def test_aggregate_to_prefix_migration_object_changes(self):
         "mysql does not support rollbacks",
     )
     def test_aggregate_to_prefix_migration_custom_fields(self):
-        self.assertEqual(self.custom_field.objects.exclude(name="example_plugin_auto_custom_field").count(), 3)
+        # This change is necessary because name attribute is not specified now in example_plugin's signal.py
+        self.assertEqual(self.custom_field.objects.exclude(name__exact="").count(), 3)
```

Comment:
Not a huge deal but `name=` and `name__exact=` are the same thing.

```suggestion
        self.assertEqual(self.custom_field.objects.exclude(name="").count(), 3)
```